cli/internal/scope/filter/filter.go

```diff
@@ -17,7 +17,9 @@ type SelectedPackages struct {
 	unusedFilters []*TargetSelector
 }
 
-type PackagesChangedSince = func(since string) (util.Set, error)
+// PackagesChangedSince is the signature of a function to provide the set of
+// packages that have changed in a particular range of git refs.
+type PackagesChangedSince = func(since string, toCommit string) (util.Set, error)
```

Comment:
Can we do `fromRef` and `toRef` for all the places and things?

```
since => fromRef
diffBase(.*) => fromRef$0
diffHead(.*) => toRef$0
```

Reply:
Did the rename. Kept references to legacy `--since` specifically for the `LegacyFilter`, but everything else is now refs.